AmericanMedicalAssociation/AMA-style-guide
# Ama 26 hero cta headline

https://palantir.atlassian.net/browse/AMA-26

Adds the hero cta headline, updates the gray color variables to hex to work without Compass, adds the 02-block-hero-cta.twig file for the molecule, adds global styling classes.

Steps to test:

1. Navigate to Atoms > Text > H5.
2. Observe that there is an h5
3. Navigate to Molecules > Blocks > Block Hero CTA
4. Observe the text portion of the CTA with the Highlight text and Title text on a gray background.
